templates/careers/product.html

```diff
@@ -0,0 +1,95 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %} https://docs.google.com/document/d/12rAis8nzWUOun0ZEK_L-ZGz1eEMzBgfc7UhN5t9OpJM/ {% endblock meta_copydoc %}
+
+{% block title %}Product | Careers{% endblock %}
+
+{% block meta_description %} Product management at Canonical is all about the excellence, quality and thought-leadership required to put us squarely into the frame for companies thinking afresh about their biggest technology questions. {% endblock %}
+
+{% block content %}
+<section class="p-strip--dark is-deep" style="background-position: 100% 0%; background-size: cover; background-image:url('https://assets.ubuntu.com/v1/78c0cbe3-05_suru2_dark_2K.jpg')">
+  <hr class="is-fixed-width u-no-margin--bottom is-dark">
+  <div class="row">
+    <div class="col-6">
+      <h1 class="p-heading--2"><strong>Product</strong></h1>
+    </div>
+    <div class="col-6">
+      <p class="u-sv2">In Product Management, we predict, define, deliver, and promote the coming waves of open source technology. Product Management is both an art and a science &ndash; full of creativity, problem solving, organisation, and leadership. Your experience, instincts, user focus, analytical nature and passion ensure that people across the company look to you as a leader.</p>
```

Comment:
```suggestion
      <p class="u-sv2">In Product Management, we predict, define, deliver, and promote the coming waves of open source technology. Product Management is both an art and a science &mdash; full of creativity, problem solving, organisation, and leadership. Your experience, instincts, user focus, analytical nature and passion ensure that people across the company look to you as a leader.</p>
```
This should be an em dash, as it is used to separate information in a sentance, [reference](https://www.scribbr.com/language-rules/dashes/#:~:text=The%20shorter%20en%20dash%20(%E2%80%93),a%20break%20in%20a%20sentence.)